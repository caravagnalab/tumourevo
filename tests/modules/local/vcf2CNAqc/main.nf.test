nextflow_process {

    name "Test Process VCF_PROCESSING"
    script "modules/local/vcf2CNAqc/main.nf"
    process "VCF_PROCESSING"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // meta = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                // input[0] = file("test-file.txt")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
