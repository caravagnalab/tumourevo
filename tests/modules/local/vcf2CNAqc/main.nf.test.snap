{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    [
                        {
                            "id": "MSeq_Set06_Set6_42",
                            "dataset": "MSeq",
                            "patient": "Set06",
                            "tumour_sample": "Set6_42",
                            "normal_sample": "54_N",
                            "cancer_type": "PANCANCER"
                        },
                        "MSeq_Set06_Set6_42_snv.rds:md5,8558004544b81aac9e12989aae6406c7"
                    ]
                ],
                "rds": [
                    [
                        {
                            "id": "MSeq_Set06_Set6_42",
                            "dataset": "MSeq",
                            "patient": "Set06",
                            "tumour_sample": "Set6_42",
                            "normal_sample": "54_N",
                            "cancer_type": "PANCANCER"
                        },
                        "MSeq_Set06_Set6_42_snv.rds:md5,8558004544b81aac9e12989aae6406c7"
                    ]
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-09-16T12:42:42.780949796"
    }
}